### PR TITLE
Added No Data To Display Module

### PR DESCRIPTION
**Pull request by chetansatpute**

### CODE REVIEW

**Review by kevaldalasaniya — APPROVED** [review]

File: build.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ const cdnScriptsOptional = {
   '{{version}}/modules/coloraxis.js': 1,
   '{{version}}/modules/venn.js': 1,
   "{{version}}/modules/dumbbell.js": 1,
-  '{{version}}/modules/lollipop.js': 1
+  '{{version}}/modules/lollipop.js': 1,
+  '{{version}}/modules/no-data-to-display.js' : 1
```
Comment (kreintjes) [inline build.js:66]:
```suggestion
  '{{version}}/modules/lollipop.js': 1
```

I believe a module shouldn't be added both to the Optional list as well as the Common list. It should either be optional or always included. Not sure, but I think the module is now included/downloaded twice.

I have created a PR to fix this (which also includes another module I needed): https://github.com/highcharts/node-export-server/pull/296

Comment (chetansatpute) [reply build.js:66]: sure, you can merge that one.
So we can close this one.

Comment (kreintjes) [reply build.js:66]: Thanks for the quick response. Can you close this PR? I cannot close PRs of another author.

**Comment by chetansatpute** [comment]
https://github.com/highcharts/node-export-server/pull/296
This changes will be done in above PR, so closing this one @kreintjes

**Pull request closed without merging**